tests/filecheck/dialects/gpu/ops.mlir

```diff
@@ -139,10 +139,10 @@ builtin.module attributes {"gpu.container_module"} {
 // CHECK-NEXT:             }) : (index) -> index
 
 // CHECK-NEXT:             "gpu.launch"(%{{.*}}, %{{.*}}, %{{.*}}, %{{.*}}, %{{.*}}, %{{.*}}) <{"operandSegmentSizes" = array<i32: 0, 1, 1, 1, 1, 1, 1, 0>}> ({
-// CHECK-NEXT:             ^{{.*}}(%{{.*}} : index, %{{.*}} : index, %{{.*}} : index,
-// CHECK-SAME:                 %{{.*}} : index, %{{.*}} : index, %{{.*}} : index,
-// CHECK-SAME:                 %{{.*}} : index, %{{.*}} : index, %{{.*}} : index,
-// CHECK-SAME:                 %{{.*}} : index, %{{.*}} : index, %{{.*}} : index):
+// CHECK-NEXT:             ^{{\S+}}(%{{\S+}} : index, %{{\S+}} : index, %{{\S+}} : index,
+// CHECK-SAME:                 %{{\S+}} : index, %{{\S+}} : index, %{{\S+}} : index,
+// CHECK-SAME:                 %{{\S+}} : index, %{{\S+}} : index, %{{\S+}} : index,
+// CHECK-SAME:                 %{{\S+}} : index, %{{\S+}} : index, %{{\S+}} : index):
```

Comment:
(This one is just OOC, I don't mind as this doesn't influence anything actually checked, so please ignore if you don't have time for it)

Those didn't fail, did they ?

Reply:
Nope, the current `filecheck` happily passed this check as good!